packages/server-render/server-sink.js

```diff
@@ -33,6 +35,44 @@ export class ServerSink {
     this.appendToElementById(id, html);
   }
 
+  updateHeadElementByTag(
```

Comment:
You'll need to find a way to make these changes work for the `client-sink.js` as well, so that rendering can remain server/client agnostic. That's part of why the sink API is so limited: it can only support operations that work on both the server and the client in the same way.